lib/pymedphys/docs/users/ref/lib/electronfactors.rst

```diff
@@ -0,0 +1,28 @@
+#######################
+Electron Cutout Factors
+#######################
+
+*******
+Summary
+*******
+
+.. automodule:: pymedphys.electronfactors
+    :no-members:
+
+
+
+***
+API
+***
+
+.. autofunction:: pymedphys.electronfactors.parameterise_insert
```

Comment:
Why two different spellings, and will that impact the autofunction documentation?

Reply:
Thanks Stuart :) The old spelling was a spelling mistake. So in restoring the API I took the opportunity to fix the spelling mistake in the API. But didn't do it in the docs. I will do that in a new PR.

Thank you for picking that up in review!

Reply:
...okay... so it looks like both spellings are okay. ... I will revert to the original API spelling to minimise issues.

Reply:
So, in my early Sunday morning stupor, I committed straight to `main`. I need to make it so that's not possible for me to do that. Nevertheless, I fixed the issue in the following commit:

https://github.com/pymedphys/pymedphys/commit/28006b38066de913d46c3d221288d8c28edb8044

Also, that spelling mistake did raise a warning in the docs. We need to get the docs to the point where it has now warnings, and then we can have new warnings be raised as an error.

So, two process problems highlighted here, first, Simon able to commit to main, second, docs not raising errors within the CI.